sjk-core/src/test/java/org/gridkit/jvmtool/cmd/StackSampleAnalyzerCmdTest.java

```diff
@@ -0,0 +1,45 @@
+package org.gridkit.jvmtool.cmd;
+
+import org.gridkit.jvmtool.cmd.StackSampleAnalyzerCmd.DateFormater;
```

Comment:
Oups, are there really typos in their class names?

Reply:
Yes, that's right.